ChatApp/static/css/login.css

```
@@ -0,0 +1,90 @@
+/* ログイン画面のCSS */
+
+.container {
+  background-color: #E3F8FF;
+}
+
+
+/* -------------login-form------------------------------- */
+.registration-form-wrapper {
```

他がスネークケースになっているので、一応統一しましょう！
```suggestion
.registration_form_wrapper {
```